tools/changelogger-release.sh

```diff
@@ -219,7 +219,7 @@ cat <<-EOM
 
 	  git diff '**/CHANGELOG.md'
 
-	Feel free to edit them as needed. Then commit and push a PR, and have it merged.
+	Feel free to edit them as needed. Then commit and push those changes.
 
 EOM
 
```

Comment:
Do we need this in here?

```
if [[ "$SLUG" == plugins/* ]]; then
        cd "$BASE"
        VER=
        if [[ -x "projects/$SLUG/vendor/bin/changelogger" ]]; then
                VER=$(cd "projects/$SLUG" && vendor/bin/changelogger version current)
        fi
        if [[ -n "$VER" ]]; then
                cat <<-EOM
                        When ready, you can create the release branch with

                          tools/create-release-branch.sh $SLUG ${VER%-beta}

                EOM
        fi
fi
```

The output I saw yesterday and testing for me again today is the following:

```
When ready, you can create the release branch with

  tools/create-release-branch.sh plugins/search 1.3.1
  ```

  I'm not sure why search is displaying, however I'm not sure we need this at all.